.github/workflows/autodocs-images.yaml

```diff
@@ -17,40 +17,53 @@ jobs:
       contents: write
     runs-on: ubuntu-latest
     steps:
-      - name: "Check out Destination Repo"
+      - name: Check out Destination Repo
         uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
         with:
           path: edu
 
-      - name: "Check out Images Monorepo"
+      - name: Check out Images Monorepo
         uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
         with:
           repository: chainguard-images/images
           path: images
 
-      - name: 'Setup gitsign'
+      - name: Set up gitsign
         uses: chainguard-dev/actions/setup-gitsign@main
 
-      - name: "Set up workdir"
+      - name: Set up workdir
         run: |
           mkdir -p "${{ github.workspace }}/${{ env.WORKDIR }}" && \
           chmod 777 "${{ github.workspace }}/${{ env.WORKDIR }}"
 
-      - name: "Copy current changelog to workdir"
+      - name: Set up Chainctl
+        uses: chainguard-dev/actions/setup-chainctl@main
+        with:
+          # This identity allows us to list the public Chainguard images.
+          identity: 720909c9f5279097d847ad02a2f24ba8f59de36a/a033a6fabe0bfa0d
```

Comment:
Should this be an environment variable in the repo, or a secret?

Reply:
+1 variable